system-parachains/coretime/coretime-kusama/src/lib.rs

```diff
@@ -244,6 +244,7 @@ impl pallet_authorship::Config for Runtime {
 
 parameter_types! {
 	pub const ExistentialDeposit: Balance = SYSTEM_PARA_EXISTENTIAL_DEPOSIT;
+	pub const RandomParaId: ParaId = ParaId::new(43211234);
```

Comment:
We don't need to expose benchmarking data:
```suggestion
```

I would move this to the `parameter_types` bellow https://github.com/polkadot-fellows/runtimes/pull/483/files#diff-ffc619456a6540ece9f2311f51e8b7f4c53de43f3624d5ca5d605fab4964367aR935-R940
```
			parameter_types! {
				pub ExistentialDepositAsset: Option<Asset> = Some((
					KsmRelayLocation::get(),
					ExistentialDeposit::get()
				).into());
				pub const RandomParaId: ParaId = ParaId::new(43211234);
			}
```

(the same for coretime-polkadot)